README.md

```diff
@@ -69,6 +69,16 @@ Next, any of the following optional parameters may be specified:
   DO_THING=false
   ```
 
+* `$BUILDKIT_SECRET_*`: extra secrets which are made available via
```

Comment:
I was a bit curious as to whether to call this `BUILDKIT` or `BUILD`, since I'm trying to keep from exposing any buildkit-specific flags in this image so that the building can be swapped out with e.g. Kaniko or some other builder in the future. But it looks like the secret mounting stuff is *pretty* buildkit-specific, with the Kaniko flow being to just tuck credentials under `/kaniko` which is ignored wholesale.

Since there doesn't seem to be a general way of handling this across image building tools, I'm OK with just being upfront and calling it `BUILDKIT`. 👍🏻

Reply:
Yeah, it does feel a bit gross putting specific flags in for a given tool (buildkit, in this case) into this resource. Maybe, in the future, would it be worth having multiple tool-specific build resources, eg. `buildkit-build-task`?

Reply:
Yeah maybe - I had hoped to hide this all away so I can change out the details later, but perhaps it would have been easier to just be explicit about the toolchain. Food for thought for when we convert this to a Prototype. :)